pkg/authorizer/server/server.go

```diff
@@ -79,7 +79,7 @@ func (a *Authorizer) AuthorizeHTTP(w http.ResponseWriter, req *http.Request) {
 	userToken := auth[1]
 
 	var (
-		resp *TokenResponse
+		resp *ClusterRegistration
 		err  error
 	)
 	if a.to == nil {
```

Comment:
this adhoc polymorphism is a little funny. it would be cleaner to configure the Authorizer struct with an authorize func rather than switching different funcs based on the `to` field. we can clean this up in a followup

Reply:
yes, totally agreed, I wanted this factoring to be least invasive, merely reflecting necessary changes towards tollbooth, otherwise the review would have been messier.

I have also many ideas how to move forward with this source code base and we definitely should move forward with this once we have the interface with tollbooth straight.